language/extensions/async/move-async-vm/tests/sources/AccountStateMachine.exp

```diff
@@ -5,10 +5,10 @@ publishing vector
 publishing AccountStateMachine
 actor 0x4 created from 0x3::AccountStateMachine
   SUCCESS
-  commit 0x3::AccountStateMachine::AccountStateMachine[0x4] := [00, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00]
+  commit 0x3::AccountStateMachine::AccountStateMachine[0x4] := New("[00, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00]")
```

Comment:
@wrwg let me know if you want me to change the formatting here.

Reply:
Thats fine, it is just kind of debugging output.